### PR TITLE
Issue/607

### PR DESCRIPTION
**Pull request by jccosta**
Fixes the billboard alignment problems when using fisheye projection.

### CODE REVIEW

File: modules/digitaluniverse/rendering/renderablebillboardscloud.cpp [hunk 1]
```diff
@@ -204,15 +207,31 @@ namespace {
...
 }  // namespace
 
 namespace openspace {
 
-documentation::Documentation RenderableBillboardsCloud::Documentation() {
-    using namespace documentation;
-    return {
-        "RenderableBillboardsCloud",
-        "digitaluniverse_RenderableBillboardsCloud",
+    documentation::Documentation RenderableBillboardsCloud::Documentation() {
```
Comment (alexanderbock) [inline modules/digitaluniverse/rendering/renderablebillboardscloud.cpp]: Indentation in the entire file

File: modules/digitaluniverse/shaders/billboard_gs.glsl [hunk 2]
```diff
@@ -101,91 +85,94 @@ void main() {
...
-    texCoord    = corners[3];
-    gl_Position = thirdPosition;
+    texCoord    = corners[1];
+    gl_Position = secondPosition;
     EmitVertex();
-    EndPrimitive(); // Second Triangle    
-}
+    EndPrimitive();     
+
+}
```
Comment (alexanderbock) [inline modules/digitaluniverse/shaders/billboard_gs.glsl]: No newline at the end of the file

File: modules/digitaluniverse/shaders/billboard_gs.glsl [hunk 3]
```diff
@@ -27,21 +27,26 @@
...
 
 layout(points) in;
-layout(triangle_strip, max_vertices = 6) out;
+layout(triangle_strip, max_vertices = 4) out;
 
-//uniform dmat4 transformMatrix;
-uniform dmat4 modelViewProjectionTransform;
 uniform float scaleFactor;
-uniform dvec3 up;
+uniform dvec3 up;
```
Comment (alexanderbock) [inline modules/digitaluniverse/shaders/billboard_gs.glsl]: Extra spaces at the end

File: modules/digitaluniverse/shaders/billboard_gs.glsl [hunk 4]
```diff
@@ -101,91 +85,94 @@ void main() {
     } else if (pos.w == 6.f) {
         unit = 306391534.73091 * PARSEC;
```
Comment (alexanderbock) [inline modules/digitaluniverse/shaders/billboard_gs.glsl]: I know it's not your code, but if you know, could you put a comment where that number comes from?

File: modules/digitaluniverse/shaders/billboard_fs.glsl [hunk 5]
```diff
@@ -53,17 +53,18 @@ Fragment getFragment() {
...
+    if (fullColor.a == 0.f || 
+        fullColor.rgb == vec3(0.0)) {
         discard;
     }
 
     Fragment frag;
     frag.color      = fullColor;
     frag.depth      = vs_screenSpaceDepth;
-    frag.gPosition  = vec4(1e32, 1e32, 1e32, 1.0);
+    frag.gPosition  = vec4(-1e32, -1e32, -1e32, 1.0);
```
Comment (alexanderbock) [inline modules/digitaluniverse/shaders/billboard_fs.glsl]: A comment explaining these numbers?

File: modules/digitaluniverse/shaders/billboard_gs.glsl (hunk 2 above)
Comment (jccosta) [reply modules/digitaluniverse/shaders/billboard_gs.glsl]: Done.

File: modules/digitaluniverse/shaders/billboard_gs.glsl (hunk 3 above)
Comment (jccosta) [reply modules/digitaluniverse/shaders/billboard_gs.glsl]: Done.
I need to change my editor. :-)

File: modules/digitaluniverse/shaders/billboard_gs.glsl (hunk 4 above)
Comment (jccosta) [reply modules/digitaluniverse/shaders/billboard_gs.glsl]: Done and shamefully it is my code. :-D

File: modules/digitaluniverse/shaders/billboard_fs.glsl (hunk 5 above)
Comment (jccosta) [reply modules/digitaluniverse/shaders/billboard_fs.glsl]: Done.

File: modules/digitaluniverse/rendering/renderablebillboardscloud.cpp (hunk 1 above)
Comment (jccosta) [reply modules/digitaluniverse/rendering/renderablebillboardscloud.cpp]: I didn't get it. What is the problem here? How should it be?

**Pull request merged**